### PR TITLE
Update README.md to explain installing triton

### PR DESCRIPTION
**Pull request by wconstab**
users building from source should know how to install triton the recommended way.

### CODE REVIEW

**Review by msaroufim — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -187,6 +187,8 @@ Other potentially useful environment variables may be found in `setup.py`.
 conda install cmake ninja
 # Run this command from the PyTorch directory after cloning the source code using the “Get the PyTorch Source“ section below
 pip install -r requirements.txt
+# (optional) If using torch.compile with inductor/triton, install the matching version of triton
+make triton
```
Comment (alanwaketan) [inline README.md:191]: Is this platform independent?

**Review by alanwaketan — APPROVED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (msaroufim) [reply README.md:191]: it'll only work on linux, willl break on windows and mac

Comment (alanwaketan) [reply README.md:191]: Then should we move this to the Linux section below?

Comment (wconstab) [reply README.md:191]: oh i didn't realize triton is linux only.

**Pull request closed without merging**